### PR TITLE
Adding npm version badge

### PR DESCRIPTION
**Pull request by nikithauc**

### CODE REVIEW

**Review by baywet — CHANGES_REQUESTED** [review]
FYI I'm using shields.io which is kind of a standard to do those sort of things on the Java SDK.
It'd be nice if we could align on a provider across the SDKs (look and feel, reducing dependencies, etc...)
https://github.com/microsoftgraph/msgraph-sdk-java/blob/dev/README.md

https://img.shields.io/npm/v/@microsoft/microsoft-graph-types-beta.svg?style=flat

**Comment by baywet** [comment]
this is also what msal is using FYI https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/README.md

File: README.md [hunk 1]
```diff
@@ -25,9 +27,8 @@ import { Client } from "@microsoft/microsoft-graph-client";
 import 'isomorphic-fetch';
 
 const client = Client.initWithMiddleware({
-	debugLogging: true,
-	middleware,
 	defaultVersion: 'beta',
+	...
```
Comment (zengin) [inline README.md:31]: Is this an intended change?

**Review by zengin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nikithauc) [reply README.md:31]: Yes. Suggestion by @MIchaelMainer

Comment (zengin) [reply README.md:31]: ok, just wanted to make sure because I couldn't relate it to the PR title.

**Review by zengin — APPROVED** [review]

**Pull request merged**